x-pack/legacy/plugins/siem/package.json

```diff
@@ -5,6 +5,7 @@
   "private": true,
   "license": "Elastic-License",
   "scripts": {
+    "extract-mitre-attacks": "node scripts/extract_tactics_techniques_mitre.js & node ../../../../scripts/eslint ./public/pages/detection_engine/mitre/mitre_tactics_techniques.ts --fix",
```

Comment:
This is absolutely awesome @XavierM! Thanks for taking the time to make a re-usable script for updating these -- our future selves are thanking you graciously! 😅 I tested this with both removing some fields from the current and deleting the file altogether and looks good! When deleting the file for a clean re-gen it'll give a no matching pattern error when trying to lint (and you'll have to lint manually), but it still generates fine. (Not a big deal since we'll be generating with the file in tact.)